.buildkite/scripts/common.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -euo pipefail
+
+WORKSPACE="$(pwd)/bin"
+
+add_bin_path(){
+    mkdir -p ${WORKSPACE}
+    export PATH="${WORKSPACE}:${PATH}"
+}
+
+with_go() {
+    mkdir -p ${WORKSPACE}
+    retry 5 curl -sL -o ${WORKSPACE}/gvm "https://github.com/andrewkroh/gvm/releases/download/${SETUP_GVM_VERSION}/gvm-linux-amd64"
+    chmod +x ${WORKSPACE}/gvm
+    eval "$(gvm $(cat .go-version))"
+    go version
+    which go
+    export PATH="$(go env GOPATH):$(go env GOPATH)/bin:${PATH}"
+    echo -e "\nPATH="${PATH}"" >> dev-tools/integration/.env
+}
+
+with_docker_compose() {
+    mkdir -p ${WORKSPACE}
+    retry 5 curl -SL -o ${WORKSPACE}/docker-compose "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-linux-x86_64"
+    chmod +x ${WORKSPACE}/docker-compose
+    docker-compose version
+    export PATH="${WORKSPACE}:${PATH}"
+    echo -e "\nPATH="${PATH}"" >> dev-tools/integration/.env
```

Comment:
Why do we need to change the `dev-tools/integration/.env`?
Looking at the existing Jenkinsfile this env is something that is required to be available as env vars inside the docker-compose containers, why we had to add the path to the workspace?